io-engine/src/bdev/nexus/nexus_bdev_rebuild.rs

```diff
@@ -127,6 +132,11 @@ impl<'n> Nexus<'n> {
         // rebuilt ranges in sync with the other children.
         self.reconfigure(DrEvent::ChildRebuild).await;
 
+        // Drop rebuild log from the child.
+        if let Ok(c) = self.as_mut().child_mut(&dst_child_uri) {
+            c.drop_rebuild_log()
```

Comment:
What if the rebuild fails, does this mean we lose the log and have to fully rebuild?

Reply:
Yes. I think it actually makes sense as if a (partial) rebuild fails, something fishy is already going on.